Sources/XCTest/XCPredicateExpectation.swift

```diff
@@ -0,0 +1,53 @@
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//
+//  XCPredicateExpectation.swift
+//  Expectations with a specified predicate and object to evaluate.
+//
+
+#if os(Linux) || os(FreeBSD)
+    import Foundation
+#else
+    import SwiftFoundation
+#endif
+
+public class XCPredicateExpectation: XCTestExpectation {
+    internal let predicate: NSPredicate
+    internal let object: AnyObject
+    internal var timer: NSTimer?
+    internal let handler: XCPredicateExpectationHandler?
+    private let evaluationInterval = 1.0
```

Comment:
An evaluation interval of one second seems super slow to me... is similar to the evaluation interval for Apple XCTest? I would mimic Apple XCTest, but if I had to choose an interval I would choose something closer to `0.1` second, or maybe even a shorter interval than that!